tests/view/map_input_to_commands.rs

```diff
@@ -255,6 +255,7 @@ fn should_parse_number_json() {
         "treq",
         "GET",
         "url.com",
+        "-b",
         "Hello=World",
         "age:=29",
         "amount:=-30.8",
```

Comment:
Create another tests just to test the "-b" flag. Event though you replicate almost the same tests, putting this "-b" in input, if these tests fail we can easily find the features which has broke (instead thinking the error is another place because the "should_parse_number_json" test has failed.